src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -63,10 +67,21 @@ public ListPanel(UI ui, GUIController guiController, PanelControl parentPanelCon
         super(ui, guiController, parentPanelControl, panelIndex);
         this.ui = ui;
         this.guiController = guiController;
-
+        totalLabel = new Label("Total : - ");
```

Comment:
This this better be inside of the `createPanelFooter` method? Since it's a subcomponent of the footer.